### DISCUSSION TITLE
Suggested Logo (Draft)

### DISCUSSION THREAD

**Discussion by inbal2l** [bemanproject/beman#10] [Ideas]
![Beman_Icom_png](https://github.com/beman-org/beman/assets/12187340/128adf02-a968-44e6-9847-b97dd8c1b6c5)

**Comment by camio** [comment]
Reasonable for a temporary logo. Pretty ugly though IMO :-D

**Reply by inbal2l** [reply]
IMO it have just the right combination of pretty and ugly. 😆
But it's good that you speak up, as this community encorag variety of opinions (even wrong ones 😋)

**Reply by JeffGarland** [reply]
I like Brett's idea of stylizing it through some AI thing

**Reply by inbal2l** [reply]
Sounds good! I liked the idea of the image, but any digital editing is welcome. I'll send the sources to you, Bret, and David.

**Comment by sakshamsharma** [comment]
I decided to try some AI based approaches to refine this logo. I'll keep iterating but here's where I am right now if that helps / people have any directional comments.
![IMG_1921](https://github.com/beman-project/beman/assets/10418596/b7a52cce-5134-4db8-b1f2-2fac1fa7069a)

**Reply by JeffGarland** [reply]
I like it -- we should probably move the discussion to discourse...
